src/Nest/QueryDsl/FullText/Intervals/IntervalsPrefix.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Linq.Expressions;
+using System.Runtime.Serialization;
+
+namespace Nest
+{
+	/// <summary>
+	/// matches terms that start with a specified set of characters. This prefix can expand to match at most 128 terms.
+	/// If the prefix matches more than 128 terms, Elasticsearch returns an error.
+	/// You can use the index-prefixes option in the field mapping to avoid this limit.
+	/// <para />
+	/// Available in Elasticsearch 7.3.0+
+	/// </summary>
+	[ReadAs(typeof(IntervalsPrefix))]
+	public interface IIntervalsPrefix : IIntervalsNoFilter
+	{
+		/// <summary>
+		/// Analyzer used to normalize the prefix. Defaults to the top-level field's analyzer.
+		/// </summary>
+		[DataMember(Name = "analyzer")]
+		string Analyzer { get; set; }
+
+		/// <summary>
+		/// Beginning characters of terms you wish to find in the top-level field
+		/// </summary>
+		[DataMember(Name = "prefix")]
+		string Prefix { get; set; }
+
+		/// <summary>
+		/// If specified, then match intervals from this field rather than the top-level <field>.
+		/// The prefix is normalized using the search analyzer from this field, unless a separate analyzer is specified.
+		/// </summary>
+		[DataMember(Name = "use_field")]
+		Field UseField { get; set; }
+	}
+
+	/// <inheritdoc cref="IIntervalsPrefix" />
+	public class IntervalsPrefix : IntervalsNoFilterBase, IIntervalsPrefix
+	{
+		/// <inheritdoc />
+		[DataMember(Name = "analyzer")]
+		public string Analyzer { get; set; }
+
+		/// <inheritdoc />
+		public string Prefix { get; set; }
```

Comment:
Missing `DataMember`?

Reply:
Thanks for catching this. The concrete impl doesn't need the attributes. Will update